Fixed return code in _times syscall.

divan0v opened the pull request:
In success case syscall returns zero.

divan0v committed:
Fixed return code in _times syscall.

aswaterman commented:
I don't think zero is correct, either: http://pubs.opengroup.org/onlinepubs/009695399/functions/times.html

Shouldn't it return something equivalent to `utime * sysconf(_SC_CLK_TCK) / 1000000`?

divan0v committed:
Fixed return value in _times syscall. Upon successful completion,
times() shall return the elapsed real time, in clock ticks, since
an arbitrary point in the past (for example, system start-up time).

divan0v commented:
I agree with your comments. Please, check corrected version.

kito-cheng commented:
LGTM, thanks!
